app-ios/Sources/TimetableFeature/TimetableDataItems.swift

```diff
@@ -27,6 +28,12 @@ public struct TimetableTimeGroupItems: Identifiable, Equatable, Hashable {
         self.endsTimeString = endsTimeString
         self.items = items
     }
+    
+    func getItemForRoom(forRoom: RoomType) -> TimetableItemWithFavorite? {
```

Comment:
how about this? 👀 
```suggestion
    func getItem(for room: RoomType) -> TimetableItemWithFavorite? {
```

this is mentioned in [Swift API Guidelines](https://www.swift.org/documentation/api-design-guidelines/#naming) `Omit needless words`